BeSpoke/Views/NewEntryView.swift

```diff
@@ -0,0 +1,56 @@
+//
+// NewEntryView.swift
+// bespoke
+//
+// Created by Ethan Wright on 2023-12-11
+//
+        
+
+import Foundation
+import FirebaseCore
+import FirebaseFirestore
+import SwiftUI
+
+struct NewEntryView: View {
+    @State private var title = ""
+    @State private var content = ""
+    @State private var submitted = false
+
+    var body: some View {
+        VStack {
+            TextField("Title", text: $title)
+                .textFieldStyle(DefaultTextFieldStyle())
+                .padding()
+
+            TextField("Journal Entry", text: $content)
+                .textFieldStyle(DefaultTextFieldStyle())
+                .padding()
+
+            NavigationLink(destination: ScrollingView(), isActive: $submitted) {
+                EmptyView()
+            }
+            .hidden()
+            
+            Button("Submit") {
+                do {
+                    let db = Firestore.firestore()
+                    
+                    // TODO: get current date and location
```

Comment:
We'll need some way of hiding/not using location if user doesn't allow location services access

Reply:
Yeah didn't wanna tackle location services in this PR, needs some more research. Opened #15.